modules/widgets/mailchimp.php

```diff
@@ -32,28 +59,125 @@ function __construct() {
 					'customize_selective_refresh' => true,
 				)
 			);
+			// These are the same attributes as the Mailchimp form (extensions/blocks/mailchimp/mailchimp.php), so if the block attributes are changed the these need to be changed accordingly.
```

Comment:
Do you think you could add the same comment in `extensions/blocks/mailchimp/mailchimp.php`, since that's most likely where most changes would initiate?

Reply:
I added a note to the `load_assets` function where I warn about the same attrbiutes in the Mailchimp Widget.